internal/worker/httpserverargs/worker.go

```diff
@@ -85,9 +85,16 @@ func newWorker(ctx context.Context, cfg workerConfig) (worker.Worker, error) {
 		return nil, errors.Trace(err)
 	}
 
+	st, err := w.cfg.statePool.SystemState()
```

Comment:
**suggestion:** Can we add a comment stating that this will be the controller model uuid.

Reply:
Instead I have renamed variables to make this clearer